src/main/resources/application-c100-transform.yaml

```diff
@@ -164,3 +164,16 @@ uk:
                     canYouProvideEmailAddress: respondent2EmailAddressDontKnow
                     isAtAddressLessThan5Years: respondent2LivedAtThisAddressForOverFiveYears
                     addressLivedLessThan5YearsDetails: respondent2AllAddressesForLastFiveYears
+              checklistSection1Completed: section1RelatingToChildCompleted
+              checklistSection2To4Completed: section2Section3Section4RelatingToMediationCompleted
+              checklistSection5&6Completed: section5Section6RelatingToApplicationReasonCompleted
+              checklistSection7Completed: section7RelatingToOtherCourtFullyCompleted
+              checklistSection8To10Completed: section8Section9Section10FactorsAffectingProceedingsCompleted
+              checklistSection11Completed: section11RelatingToApplicantCompleted
+              checklistSection12To14Completed: sections12Section13Section14RelatingToRespondentAndOthersNoticeCompleted
+              checklistSection16Completed: section16RelatingToTruthStatementCompleted
+              printFullName: printFullName
+              agreedBy: applicantSolicitorsFirmName
+              SignedDate: applicantAndApplicantsolicitorSignedDate
+              positionOrOfficeHeld: position_or_OfficeHeld
+              courtFee: courtFee
```

Comment:
can you change  to courtFeeConsent instead of courtFee?

Reply:
This has been changed